src/style.css

```diff
@@ -1 +1,107 @@
-/* add styles here */
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face{
+  font-family: Avenir;
+  src:url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --basic-color: #3d4e61;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md):
- Don't use * selector (it impacts performance)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/63243136/45ce3530-5d7a-4ff3-89f9-c2dfd5909506)